StarWars.Core/ViewModels/MainViewModel.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Collections.ObjectModel;
+using System.Threading.Tasks;
+using MvvmCross.Commands;
+using MvvmCross.Navigation;
+using MvvmCross.ViewModels;
+using StarWars.Core.Models;
+using StarWars.Core.Services.Abstract;
+
+namespace StarWars.Core.ViewModels;
+
+public sealed class MainViewModel : MvxViewModel
+{
+    private readonly IMvxNavigationService _navigationService;
+    private readonly ISwapiService _swapiService;
+    public IMvxCommand<SwapiPersonModel> SelectPersonCommand { get; }
+
+    public MainViewModel(IMvxNavigationService navigationService, ISwapiService swapiService)
+    {
+        _navigationService = navigationService;
+        _swapiService = swapiService;
+        
+        SelectPersonCommand = new MvxCommand<SwapiPersonModel>(async person =>
+        {
+            if (person != null)
+            {
+                await navigationService.Navigate<DetailViewModel, SwapiPersonModel>(person);
+            }
+        });
+    }
+    
+    private ObservableCollection<SwapiPersonModel> _people;
+    public ObservableCollection<SwapiPersonModel> People
```

Comment:
Here is more optimal to create like that:
`public ObservableCollection<SwapiPersonModel> People { get; } = new ();`